src/components/ExpensiTextInput/ExpensiTextInputLabel/expensiTextInputLabelPropTypes.js

```diff
@@ -3,13 +3,23 @@ import {Animated} from 'react-native';
 
 const propTypes = {
     /** Label */
-    label: PropTypes.string,
+    label: PropTypes.string.isRequired,
 
     /** Label vertical translate */
     labelTranslateY: PropTypes.instanceOf(Animated.Value).isRequired,
 
     /** Label scale */
     labelScale: PropTypes.instanceOf(Animated.Value).isRequired,
+
+    /** For attribute for label (only Web) */
```

Comment:
Ok, I am just a little confused about achieving this. TextInput needs this prop which is used in BaseExpensiInput. 
Are you suggesting that
1. I make a component TextinputWrapper, native, web.
2. Then in web one, use setNativeProps to set it.

But still name prop needs to be passed through ExpensiInput all the way to web version on Web.

Reply:
>I make a component TextinputWrapper, native, web.

Not a wrapper. I was thinking to just pass the prop down, but only use it in the non native versions. We don't need to mention here that "only web" uses this. We should just make it so all `ExpensiTextInput` get the generic `for` prop, but native doesn't use it. Make sense?

Reply:
Actually, it does that. `for` is passed as `' '` for native. As `name` prop is not available on the RN TextInput we have to use `setNativeProps` so I thought to just noop  `setNativeProps` on native. 

I used "only web" so that we know it does not have any effect on native platforms. 

Thus I created `setBrowserAttribute`.

Reply:
Yeah, I think we should not have a method that refers to a specific platform that can be used anywhere. Better to set the native props for whatever the component in `index.js` or `index.web.js` files etc and have the `index.native.js` file do nothing with the prop.